packages/cli/src/commands/mock.ts

```diff
@@ -1,43 +1,101 @@
 import { Command } from '@oclif/command';
+import { createLogger, logLevels } from '@stoplight/prism-core';
+import chalk from 'chalk';
```

Comment:
Good point, this could be a good candidate to extraction when we will have new commands. I can do it now. I'd defer this when we'll have the opportunity though.

Reply:
Sounds good to me.